modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserServiceTest.java

```diff
@@ -214,6 +231,69 @@ public void testUpdateOrganizationsWithoutPermission() throws Exception {
 		}
 	}
 
+	@Test
+	public void testWorkflowTaskIndexedAfterCreatingUser() throws Exception {
+		String originalName = PrincipalThreadLocal.getName();
+
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		try {
+			PrincipalThreadLocal.setName(TestPropsValues.getUserId());
+
+			PermissionChecker adminPermissionChecker =
+				PermissionCheckerFactoryUtil.create(TestPropsValues.getUser());
+
+			PermissionThreadLocal.setPermissionChecker(adminPermissionChecker);
+
+			_workflowDefinitionLinkLocalService.addWorkflowDefinitionLink(
+				null, TestPropsValues.getUserId(),
+				TestPropsValues.getCompanyId(),
+				WorkflowConstants.DEFAULT_GROUP_ID, User.class.getName(), 0, 0,
+				"Single Approver", 1);
+
+			User user = _userService.addUserWithWorkflow(
+				TestPropsValues.getCompanyId(), true, StringPool.BLANK,
+				StringPool.BLANK, true, StringPool.BLANK,
+				"UserServiceTest." + RandomTestUtil.nextLong() + "@liferay.com",
+				LocaleUtil.getDefault(), "UserServiceTest", StringPool.BLANK,
+				"UserServiceTest", 0, 0, true, Calendar.JANUARY, 1, 1970,
+				StringPool.BLANK, null, null, null, null,
+				new ArrayList<Address>(), new ArrayList<EmailAddress>(),
+				new ArrayList<Phone>(), new ArrayList<Website>(),
+				new ArrayList<AnnouncementsDelivery>(), false,
+				ServiceContextTestUtil.getServiceContext());
+
+			WorkflowModelSearchResult<WorkflowTask> workflowModelSearchResult =
+				WorkflowTaskManagerUtil.searchWorkflowTasks(
+					TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+					StringPool.BLANK, new String[0], null, null, null, null,
+					null, null, null, true, true, null, null, false, 0, 20,
+					null);
+
+			Assert.assertEquals(1, workflowModelSearchResult.getLength());
+
+			WorkflowTask workflowTask =
+				workflowModelSearchResult.getWorkflowModels(
+				).get(
+					0
+				);
```

Comment:
can we split this in two "lines"?

Reply:
You mean like this?
```
			WorkflowTask workflowTask =
				workflowModelSearchResult.getWorkflowModels().get(0);
```
I tried but SF corrected it to this